pkg/pillar/cmd/domainmgr/domainmgr_test.go

```diff
@@ -157,3 +159,132 @@ func TestFetchEnvVariablesFromCloudInit(t *testing.T) {
 		}
 	}
 }
+
+func TestCreateMountPointExecEnvFiles(t *testing.T) {
+	content := `
+{
+  "acVersion": "1.26.0",
+  "acKind": "PodManifest",
+  "apps": [
+    {
+      "name": "foobarbaz",
+      "image": {
+        "name": "registry-1.docker.io/library/redis",
+        "id": "sha512-572dff895cc8521bcc800c7fa5224a121d3afa8b545ff9fd9c87d9c5ff090469",
+        "labels": [
+          {
+            "name": "os",
+            "value": "linux"
+          },
+          {
+            "name": "arch",
+            "value": "amd64"
+          },
+          {
+            "name": "version",
+            "value": "latest"
+          }
+        ]
+      },
+      "app": {
+        "exec": [
+          "docker-entrypoint.sh",
+          "redis-server"
+        ],
+        "user": "0",
+        "group": "0",
+        "workingDirectory": "/data",
+        "environment": [
+          {
+            "name": "PATH",
+            "value": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+          },
+          {
+            "name": "GOSU_VERSION",
+            "value": "1.11"
+          },
+          {
+            "name": "REDIS_VERSION",
+            "value": "5.0.7"
+          },
+          {
+            "name": "REDIS_DOWNLOAD_URL",
+            "value": "http://download.redis.io/releases/redis-5.0.7.tar.gz"
+          },
+          {
+            "name": "REDIS_DOWNLOAD_SHA",
+            "value": "61db74eabf6801f057fd24b590232f2f337d422280fd19486eca03be87d3a82b"
+          }
+        ],
+        "mountPoints": [
+          {
+            "name": "volume-data",
+            "path": "/data"
+          }
+        ],
+        "ports": [
+          {
+            "name": "6379-tcp",
+            "protocol": "tcp",
+            "port": 6379,
+            "count": 1,
+            "socketActivated": false
+          }
+        ]
+      }
+    }
+  ],
+  "volumes": null,
+  "isolators": null,
+  "annotations": [
+    {
+      "name": "coreos.com/rkt/stage1/mutable",
+      "value": "false"
+    }
+  ],
+  "ports": []
+}
+`
+	// create a temp dir to hold resulting files
+	dir, _ := ioutil.TempDir("/tmp", "podfiles")
+	err := os.MkdirAll(dir+"/stage1/rootfs/opt/stage2/runx", 0777)
```

Comment:
There is a lot of repetition of this path, `"/stage1/rootfs/opt/stage2/runx"`; can it be parametrized?

Reply:
Or at least put in a var?

Reply:
Good point @deitch -- fixed